types/office-js-preview/index.d.ts

```diff
@@ -96044,216 +97174,1388 @@ declare namespace Word {
-            /**
-             * Gets or sets a tag to identify a content control.
-             *
-             * @remarks
+         * Represents outline level 1.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        outlineLevel1 = "OutlineLevel1",
+        /**
+         * Represents outline level 2.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        outlineLevel2 = "OutlineLevel2",
+        /**
+         * Represents outline level 3.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        outlineLevel3 = "OutlineLevel3",
+        /**
+         * Represents outline level 4.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        outlineLevel4 = "OutlineLevel4",
+    }
+    /**
+     * Specifies the close behavior for `Document.close`.
+     *
+     * @remarks
+     * [Api set: WordApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    enum CloseBehavior {
+        /**
+         * Save the changes before closing the document.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        save = "Save",
+        /**
+         * Discard the possible changes when closing the document.
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        skipSave = "SkipSave",
+    }
+    /**
+     * Specifies the save behavior for `Document.save`.
+     *
+     * @remarks
+     * [Api set: WordApi 1.1]
+     */
+    enum SaveBehavior {
+        /**
+         * Saves the document without prompting the user. If it's a new document,
+                    it will be saved with the default name or specified name in the default location.
+         * @remarks
+         * [Api set: WordApi 1.1]
+         */
+        save = "Save",
+        /**
+         * Displays the "Save As" dialog to the user if the document hasn't been saved.
+                    Won't take effect if the document was previously saved.
+         * @remarks
+         * [Api set: WordApi 1.1]
+         */
+        prompt = "Prompt",
+    }
+    /**
+     * Represents the type of Field.
+     *
+     * @remarks
+     * [Api set: WordApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    enum FieldType {
+        /**
+         * Represents that the field type is Add-in.
```

Comment:
Would it be useful to have a brief explanation of what an Add-in field is? Or is this going to be explained in conceptual docs?

Reply:
Thx, I've added this to my backlog. support.microsoft.com has docs for many of the other field types but not this one.